src/items/itemDialog.component.tsx

```diff
@@ -1178,15 +1221,27 @@ function ItemDialog(props: ItemDialogProps) {
   };
 
   return (
-    <Dialog
-      open={open}
-      maxWidth="lg"
-      PaperProps={{ sx: { height: '810px' } }}
-      fullWidth
-    >
-      <DialogTitle>
-        {`${requestType === 'patch' ? 'Edit' : 'Add'} Item`}
+    <Dialog open={open} maxWidth="xl" fullWidth>
+      <DialogTitle sx={{ display: 'inline-flex', alignItems: 'center' }}>
+        {`${requestType === 'patch' ? 'Edit' : 'Add'} Item${isPrivilegedUser ? ' as Admin' : ''}`}
+
+        {isPrivilegedUser && (
+          <Tooltip
+            title={
+              "As an admin, you can bypass rules that restrict item placement for other users, and you can modify the item's usage status"
+            }
+            disableHoverListener={false}
```

Comment:
```suggestion
            disableHoverListener={false}
```
This is not needed as it defaults to `false`
https://mui.com/material-ui/api/tooltip/